include/swift/SIL/SILVTableVisitor.h

```diff
@@ -43,8 +43,8 @@ template <class T> class SILVTableVisitor {
   void maybeAddMethod(FuncDecl *fd) {
     assert(!fd->hasClangNode());
 
-    maybeAddEntry(SILDeclRef(fd, SILDeclRef::Kind::Func),
-                  fd->needsNewVTableEntry());
+    SILDeclRef constant(fd, SILDeclRef::Kind::Func);
+    maybeAddEntry(constant, constant.requiresNewVTableEntry());
```

Comment:
This makes me think we can just drop the second argument, since it's always recovered from the SILDeclRef now.